Makefile

```diff
@@ -53,9 +53,12 @@ install:
 
 	cp -Ra dbicdh "$(DESTDIR)"/usr/share/openqa/dbicdh
 
-test:
-	OPENQA_CONFIG= prove -r
-	./script/tidy --check
+
+.PHONY: check_style
+check_style:
```

Comment:
_ is a very untypical character for make targets. It's basically reserved for variables.